messages/deploy.metadata.md

```diff
@@ -20,41 +20,41 @@ To deploy multiple metadata components, either set multiple --metadata <name> fl
 
```

Comment:
@mshanemc , you missed one, see previous line (for some reason I can't actually click in there in the GH UI, so this comment will have to do)